subsys/bluetooth/controller/Kconfig

```diff
@@ -309,6 +312,21 @@ config BT_CTLR_TX_PWR_MINUS_40
 
 endchoice
 
+config BT_CTLR_TX_PWR_SET
+	int "Set TX power (dBm)"
+	range -127 127
+	default 0
+	depends on BT_CTLR_TX_PWR_0 && !BT_LL_SW_SPLIT
```

Comment:
```suggestion
	depends on !BT_LL_SW_SPLIT
```

Reply:
Should we not keep this and make them mutually exclusive?

Reply:
On further research keeping this would cause a dependency cycle.